notebooks/text_models/solutions/classify_text_with_bert.ipynb

```diff
@@ -9,9 +9,9 @@
     "# Classify text with BERT\n",
```

Comment:
<p>Remove "&nbsp;TF ops from the TF.text library"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/660/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>